corehq/apps/geospatial/templates/gps_capture.html

```diff
@@ -0,0 +1,145 @@
+                                {% trans "Username" %}
+                            {% else %}
+                                {% trans "Name" %}
+                            {% endif %}
+                        </th>
+                        <th class="col-xs-3">{% trans "Latitude" %}</th>
+                        <th class="col-xs-3">{% trans "Longitude" %}</th>
+                        <th class="col-xs-3">{% trans "Actions" %}</th>
+                    </tr>
+                </thead>
+                <tbody data-bind="foreach: dataItems">
+                    <tr>
+                        <td>
+                            <a target="_blank" data-bind="attr: {'href': url}">
+                                <span data-bind="text: name"></span>
+                            </a>
+                        </td>
+                        <td>
+                            <input class="form-control"
+                                   type="number"
+                                   step="any"
+                                   name="latInput"
+                                   placeholder="{% trans 'Enter latitude...' %}"
+                                   data-bind="value: lat, event: {change: onValueChanged}" />
+                        </td>
+                        <td>
+                            <input class="form-control"
+                                   type="number"
+                                   step="any"
+                                   name="lonInput"
+                                   placeholder="{% trans 'Enter longitude...' %}"
+                                   data-bind="value: lon, event: {change: onValueChanged}" />
+                        </td>
+                        <td>
+                            <button type="button" class="btn btn-default" data-bind="event: {click: onMapCaptureStart}">
+                                {% trans "Capture on Map" %}
+                            </button>
+                            <button type="button" class="btn btn-primary" data-bind="enable: canSaveRow, event: {click: $root.saveDataRow.bind($data)}">
+                                {% trans "Save" %}
+                            </button>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+            <div class="alert alert-info" data-bind="visible: showLoadingSpinner">
+                <i class="fa fa-spin fa-spinner"></i>
+                {% if data_type == 'user' %}
+                    {% trans 'Loading users...' %}
+                {% else %}
+                    {% trans "Loading cases..." %}
+                {% endif %}
+            </div>
+            <div class="alert alert-danger" data-bind="visible: hasError">
+                <i class="fa fa-exclamation-triangle"></i>
+                {% blocktrans %}
+                  <strong>There was an issue contacting the server.</strong>
+                  Please check your internet connection.
```

Comment:
This message will show when `hasError` evaluates to `true`, but currently `hasError` will also be set to `true` for errors that's not necessarily related to network issues (e.g. 500 request status). Do we need a more generic error message?

Reply:
I was following the same error message that we use for the paginated attendees list page in attendance tracking. This could be made more generic however, perhaps something along the lines of:  "There was an issue retreiving data from the server. Please check..."

Reply:
@zandre-eng Yeah, that sound's good.